src/main/java/games/strategy/engine/lobby/server/db/BannedMacController.java

```diff
@@ -36,21 +36,15 @@ public void addBannedMac(final String mac, final Instant banTill) {
     }
     Timestamp banTillTs = null;
     if (banTill != null) {
-      banTillTs = new Timestamp(banTill.toEpochMilli());
+      banTillTs = Timestamp.from(banTill);
```

Comment:
This way of getting the java.sql.Timestamp from an Instant is actually more accurate than the other way (nano precision), and as far as I can tell it is not affected by the Year-2038-Problem.

Reply:
The syntax seems simpler to me :+1: